bijection-avro/src/main/scala/com/twitter/bijection/avro/AvroCodecs.scala

```diff
@@ -121,10 +121,11 @@ object SpecificAvroCodecs {
     * @tparam T compiled Avro record
     * @return Injection
     */
-  def toJson[T <: SpecificRecordBase: ClassTag](schema: Schema): Injection[T, String] = {
-    val klass = classTag[T].runtimeClass.asInstanceOf[Class[T]]
-    val writer = new SpecificDatumWriter[T](klass)
-    val reader = new SpecificDatumReader[T](klass)
+  def toJson[T <: SpecificRecordBase: ClassTag]: Injection[T, String] = {
```

Comment:
Sure, it's what I suggested when we changed `toBinary`, but you guys decided it was unnecessary. I'll update the PR.